### DISCUSSION TITLE
挂载115网盘一段时间后无法访问，显示“URL有可能对网站造成安全威胁，您的访问被阻断。”

### DISCUSSION THREAD

**Discussion by JasonWUU15** [AlistGo/alist#7451] [Q&A]
之前一直是成功挂载，今天突然无法访问，运行日志显示“unexpected error"，在alist中点进网盘显示”failed get objs: failed to list objs“，后面的代码常见问题中没有案例，把代码跑出来是一张图，显示“由于您访问的URL有可能对网站造成安全威胁，您的访问被阻断。”，错误代码为405，并给出了我的访问id和访问时间。
不知道是哪里出了问题，求大神解答
<img width="726" alt="截屏2024-11-02 02 36 23" src="https://github.com/user-attachments/assets/a2a37b1f-bb77-4bef-8b47-dbabf2d0159b">

**Comment by snugduahn** [comment]
我也碰到了同样的问题，不知道是哪里的参数值没有设置好吗？

**Comment by sangchengfang** [comment]
设置分页大小，改大

**Comment by Atmospheric-Sciences** [comment]
冒昧问一下有解决办法吗？我也遇到了这个问题

**Reply by sangchengfang** [reply]
别用alist了，用openlist

**Comment by hight233** [comment]
115网盘不是把收购后的 alist 列进黑名单里了吗？

**Reply by hight233** [reply]
123 网盘也是
